### DISCUSSION TITLE
Is there feature like "if file x is changed, then run script y"?

### DISCUSSION THREAD

**Discussion by k4leg** [twpayne/chezmoi#2024] [Q&A] [answered]
For example, I have an emacs daemon that needs to be reloaded to read changed config, and to do this I need to do something like:
```sh
if [ "$(systemctl --user is-active emacs.service)" = "active" ]; then
        systemctl --user restart emacs.service
fi
```

**Comment by twpayne** [comment, marked as answer]
No, there's no such feature built in, but you can achieve this with a `run_after_` script that checks the checksum of the config file and restarts the service if the checksum has changed.
